PJ05/src/Exceptions/UserExistsException.java

```diff
@@ -2,15 +2,17 @@
 
 import Field.User;
 
-public class UserExistsException extends Exception{
+public class UserExistsException extends RequestFailedException {
     User user;
-    public UserExistsException(){}
 
-    public UserExistsException(String message){
+    public UserExistsException() {
+    }
+
+    public UserExistsException(String message) {
         super(message);
     }
 
-    public UserExistsException(User user){
+    public UserExistsException(User user) {
         this.user = user;
     }
```

Comment:
I understand that you want to provide some extra information in the exception. However, this is probably a redundant feature. If the client really wants to know about the existing user, it can just send another request.